backend/lib/endpoints/schema/organizations.js

```diff
@@ -0,0 +1,57 @@
+const S = require("fluent-schema");
+
+const createOrganizationSchema = {
+  body: S.object()
+    .prop("email", S.string().required())
+    .prop("global", S.boolean())
+    .prop("industry", S.string().required())
+    .prop("location", S.string().required())
+    .prop("name", S.string().required())
+    .prop("needs", S.array().maxItems(10).items(S.string()).required())
+    .prop("type", S.string()),
+};
+
+const getOrganizationSchema = {
+  params: S.object().prop("organizationId", S.string().required()),
+};
+
+// TODO: Maybe add search param in query string?
+const getOrganizationsSchema = {
+  querystring: S.object()
+    .prop("limit", S.integer())
+    .prop("ownerId", S.string())
+    .prop("skip", S.integer()),
+};
+
+const updateOrganizationSchema = {
+  body: S.object()
+    .prop("address", S.string())
+    .prop("androidUrl", S.string())
+    .prop("description", S.string())
+    .prop("email", S.string().required())
```

Comment:
i don't think update has required properties since these are PATCH requests that only use what's in the request?

Reply:
Right, that's a good point, I'll remove the required checks.

Reply:
Fixed in 9ced945